src/main/java/com/hivemq/adapter/sdk/api/ProtocolAdapterInformation.java

```diff
@@ -74,6 +76,38 @@ public interface ProtocolAdapterInformation {
      */
     @Nullable List<ProtocolAdapterTag> getTags();
 
+
+
+    /**
+     * A bean class that will be reflected upon by the framework to determine the structural requirements of the
+     * tag configuration associated with an adapter instance. It is expected that the bean class supplied, be marked up
```

Comment:
the end of the sentence is missing, right?

Reply:
Yep, fixed.